### DISCUSSION TITLE
tests happines-buff

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#99] [Tarea 1] [answered]
Hola, en el enunciado se habla del evento "HAPPINESS-BUFF" sin embargo en los tests es con guión bajo: "HAPPINESS_BUFF"... como debemos recibirlo??

**Comment by 4n1t4** [comment, marked as answer]
Léanlo de la siguiente forma :
string_equals(command, "HAPPINESS_BUFF")

Tomaremos en cuenta los test con "_" y no con "-".

Actualizaremos el enunciado para que quede igual
